sabre_smart_permission_examples/intkey/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name = "intkey"
+version = "0.1.0"
+authors = ["Cargill, Incorporated"]
+
+[dependencies]
+sabre-sdk = "0.1.3"
```

Comment:
Would should discuss if this should point Sabre master as pike will be updated to point at that, so it can be compatible with the new rust SDK API

Reply:
Personally, I think we should point to cargo whenever possible, but if it doesn't make sense to do so, or some functionality is needed the examples, I'm fine with changing it.